src/config/settings.py

```diff
@@ -299,6 +307,7 @@ def network_config(self) -> NetworkConfig:
 MAX_CONSOLIDATION_REQUEST_FEE: Gwei = decouple_config(
     'MAX_CONSOLIDATION_REQUEST_FEE', default=10, cast=int
 )
+MAX_WITHDRAWAL_REQUEST_FEE = decouple_config('MAX_WITHDRAWAL_REQUEST_FEE', default=10, cast=int)
```

Comment:
Please add _GWEI to all the env variables that expect Gwei as a value, similarly to `max_fee_per_gas_gwei`

Reply:
added type annotation